xapian-core/weight/bm25weight.cc

```diff
@@ -35,11 +35,23 @@ using namespace std;
 
 namespace Xapian {
 
+BM25Weight::BM25Weight(bool& bm25+)
+{
+	if(bm25+ != 0 && bm25+ != 1)
+	throw Xapian::InvalidArgumentError("Boolean input is invalid.");
```

Comment:
This looks like it isn't aligned properly; it's the body of the `if` so should be aligned further across than the `if` itself. Also, you're indenting to 8 spaces (as a TAB) rather than 4 as the rest of the file.